source/2020-09-01-ember-3-21-released.md

```diff
@@ -0,0 +1,88 @@
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.21
+
+Ember.js 3.21 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features (?)
+
+#### Deprecations (?)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.21, please review the [Ember.js 3.21.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.21.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.21
+
+#### New Features (?)
+
+#### Deprecations (?)
+
+For more details on changes in Ember Data 3.21, please review the
+[Ember Data 3.21.0 release page](https://github.com/emberjs/data/releases/tag/v3.21.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+<!--alex ignore easy-->
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or Ember Data by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.21
+
+#### New Features (?)
```

Comment:
`ember new --lang` support landed in https://github.com/ember-cli/ember-cli/pull/9259

Reply:
Tweak ember-try configuration for addons:

* drop Ember 3.12 (existing addons should consider this a breaking change) https://github.com/ember-cli/ember-cli/pull/9310
* add Ember 3.20 https://github.com/ember-cli/ember-cli/pull/9308

Reply:
Thanks! I will make sure to mention these items.